### PR TITLE
Feature portfolio

### PR DESCRIPTION
**Pull request by shogof**
### Description

This pull request implements several enhancements and fixes based on the provided code. Here's a summary of the changes:

1. **Added Google Fonts:** 🎨 Imported the Poppins font from Google Fonts to improve typography.
2. **Global Styles:** 🖌️ Set up global styles for basic layout adjustments, font family, and smooth scrolling behavior.
3. **Header Styling:** 🌟 Improved the styling of the header section, including a fixed position, background color, and logo animation.
4. **Home Section:** 🏠 Enhanced the home section with better alignment, text animations, and image transitions.
5. **Projects Section:** 🛠️ Optimized the projects section layout, added hover effects, and adjusted responsiveness for smaller screens.
6. **Contact Section:** ✉️ Styled the contact section with a gradient background, input fields, and submit button.

### Special Parts

- Implemented custom animations using CSS keyframes for various elements such as text, images, and buttons. 🚀
- Utilized media queries to ensure responsiveness across different device sizes, adjusting styles accordingly. 📱

Dear Ahmad Zamir Yousofi,
I've submitted this pull request for your review. Your feedback would be greatly appreciated. Thank you for considering my contribution.

Best regards,
Shogofa Developer

✨🙏✨

### CODE REVIEW

**Comment by ahzamir** [comment]
@shogof 

- Well done shogofa I checked your project and it looks imposing, but still your project misses a critical point which is responsiveness.
  - You have to change the sizing of your webpage in different screen sizes.
  - Right-click on your webpage then click on inspect and change the sizing to the mobile version sizing from there and check your design.
  
  
Screenshot 2024-06-04 121432

**Comment by shogof** [comment]
Hello Dear Teacher 
Thanks a lot that mention it 
I will do it

**Comment by shogof** [comment]
@ahzamir,

I wanted to express my sincere gratitude for taking the time to review my project. Your patience and hard work are greatly appreciated. Your feedback has been invaluable to me, as it has enabled me to learn many new things. 🙏

I am pleased to present to you the updated version of my portfolio project. This new iteration is responsive, ensuring compatibility with every screen and device. Additionally, I am committed to adopting a mobile-first approach in my next project. 📊

I kindly request your assistance in correcting any grammatical errors, improving the content, and adding relevant headings and embellishments.

Thank you once again for your support and guidance. 🌟

Best regards,
Shogofa Developer

**Comment by Marwa-Sadat** [comment]
@shogof Your project is nice, I hope you win 👏👏

**Review by ahzamir — APPROVED** [review]
### Hi @shogof 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your repo has professional README.md file
✔️Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ahzamir** [comment]
@shogof 

- Although you have done a great job by making your webpage responsive and make it look very nice, you repeated your code too many times to make it responsive and your CSS file is too long so try to use the MOBILE_FIRST approach for your next project design as it is very useful for clean CSS code.

- And don't forget to share your success in FB group and on your LinkedIn this is very important to show the world your skills.

File: README.md [hunk 1]
```diff
@@ -1 +1,233 @@
...
+      <!-- Projects content -->
+    </section>
+
+    <!-- Contact section -->
+    <section class="contact" id="contact">
+      <!-- Contact content -->
+    </section>
+  </body>
+</html>
+```
```
Comment (ahzamir) [inline README.md]: - Remove this section from your readme as it is very long.

File: styles.css
Comment (ahzamir) [inline styles.css:1]: - Your CSS file is too long.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    <header class="header">
+      <a href="#" rel="noopener" class="logo"
+        ><img src="images/mylogoo-removebg-preview.png" alt=""
+      /></a>
+      <nav class="navbar">
+        <a rel="noopener" data-order="1" href="#hom" class="ac">Home</a>
+        <a rel="noopener" data-order="2" href="#about">About</a>
+        <a rel="noopener" data-order="3" href="#proj">Project</a>
+        <a rel="noopener" data-order="4" href="#contact">Contact</a>
+      </nav>
```
Comment (ahzamir) [inline index.html:27]: - Your menu bar does not show up on the mobile version

**Review by ahzamir — CHANGES_REQUESTED** [review]
- [REQUIRED] - Solve the issues
- Check the comments below

File: README.md (hunk 1 above)
Comment (shogof) [reply README.md]: OK Sure Thanks

File: styles.css
Comment (shogof) [reply styles.css:1]: I will do it mobile first next time

**Pull request merged**